BootloaderCommonPkg/Include/Library/BootloaderCommonLib.h

```diff
@@ -27,6 +27,8 @@
 #define  ALIGN_UP(address, align)     (((address) + ((align) - 1)) & ~((align)-1))
 #define  ALIGN_DOWN(address, align)   ((address) & ~((align)-1))
 
+typedef  VOID  (*PLATFORM_CPU_INIT_HOOK) (VOID);
```

Comment:
Putting it into BootloaderCorePkg\Include\Library\MpInitLib.h

Reply:
Make sense. will do it.